### DISCUSSION TITLE
Welcome to w4af Discussions!

### DISCUSSION THREAD

**Discussion by security-companion** [security-companion/w3af#274] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by codders** [comment]
Hi all!

I am @codders - I have done a lot of the python2->python3 porting work over the last year. I'm excited to start using w4af in real projects, and welcome anyone who wants to collaborate and give feedback on the project.

**Reply by security-companion** [reply]
@codders Good to have you on board. Thanks for all the effort you already but into the project.

I am security-companion. I've also been helping with bringing the project back on track. But the appreciation definitively goes to codders as he's the one who has been doing the heavy lifting.
